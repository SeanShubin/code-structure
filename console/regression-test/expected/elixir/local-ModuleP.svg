<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 8.0.2 (20230410.1723)
 -->
<!-- Title: detangled Pages: 1 -->
<svg width="191pt" height="260pt"
 viewBox="0.00 0.00 190.54 260.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 256)">
<title>detangled</title>
<polygon fill="lightgray" stroke="none" points="-4,4 -4,-256 186.54,-256 186.54,4 -4,4"/>
<!-- ModuleP -->
<g id="node1" class="node">
<title>ModuleP</title>
<g id="a_node1"><a xlink:href="local-ModuleP.html" xlink:title="ModuleP">
<ellipse fill="none" stroke="black" stroke-width="2" cx="66.94" cy="-234" rx="43.59" ry="18"/>
<text text-anchor="middle" x="66.94" y="-230.3" font-family="Times,serif" font-size="14.00" fill="blue">ModuleP</text>
</a>
</g>
</g>
<!-- PathP.ModuleQ -->
<g id="node3" class="node">
<title>PathP.ModuleQ</title>
<g id="a_node3"><a xlink:href="local-PathP-ModuleQ.html" xlink:title="PathP.ModuleQ">
<ellipse fill="none" stroke="black" cx="66.94" cy="-162" rx="66.89" ry="18"/>
<text text-anchor="middle" x="66.94" y="-158.3" font-family="Times,serif" font-size="14.00" fill="blue">PathP.ModuleQ</text>
</a>
</g>
</g>
<!-- ModuleP&#45;&gt;PathP.ModuleQ -->
<g id="edge1" class="edge">
<title>ModuleP&#45;&gt;PathP.ModuleQ</title>
<path fill="none" stroke="black" d="M66.94,-215.7C66.94,-208.24 66.94,-199.32 66.94,-190.97"/>
<polygon fill="black" stroke="black" points="70.44,-191.1 66.94,-181.1 63.44,-191.1 70.44,-191.1"/>
</g>
<!-- PathI.ModuleJ -->
<g id="node2" class="node">
<title>PathI.ModuleJ</title>
<g id="a_node2"><a xlink:href="local-PathI-ModuleJ.html" xlink:title="PathI.ModuleJ">
<ellipse fill="none" stroke="black" cx="66.94" cy="-18" rx="63.09" ry="18"/>
<text text-anchor="middle" x="66.94" y="-14.3" font-family="Times,serif" font-size="14.00" fill="blue">PathI.ModuleJ</text>
</a>
</g>
</g>
<!-- PathI.ModuleJ&#45;&gt;PathP.ModuleQ -->
<g id="edge2" class="edge">
<title>PathI.ModuleJ&#45;&gt;PathP.ModuleQ</title>
<path fill="none" stroke="black" d="M54.94,-35.74C48.52,-45.74 41.27,-58.99 37.94,-72 33.98,-87.5 33.98,-92.5 37.94,-108 40.31,-117.25 44.66,-126.62 49.28,-134.85"/>
<polygon fill="black" stroke="black" points="46.78,-136.49 54.94,-143.26 52.78,-132.88 46.78,-136.49"/>
</g>
<!-- PathR.ModuleS -->
<g id="node4" class="node">
<title>PathR.ModuleS</title>
<g id="a_node4"><a xlink:href="local-PathR-ModuleS.html" xlink:title="PathR.ModuleS">
<ellipse fill="none" stroke="black" cx="114.94" cy="-90" rx="67.69" ry="18"/>
<text text-anchor="middle" x="114.94" y="-86.3" font-family="Times,serif" font-size="14.00" fill="blue">PathR.ModuleS</text>
</a>
</g>
</g>
<!-- PathP.ModuleQ&#45;&gt;PathR.ModuleS -->
<g id="edge3" class="edge">
<title>PathP.ModuleQ&#45;&gt;PathR.ModuleS</title>
<path fill="none" stroke="black" d="M78.56,-144.05C84.16,-135.89 91,-125.91 97.24,-116.82"/>
<polygon fill="black" stroke="black" points="100.66,-119.02 103.43,-108.79 94.89,-115.06 100.66,-119.02"/>
</g>
<!-- PathR.ModuleS&#45;&gt;PathI.ModuleJ -->
<g id="edge4" class="edge">
<title>PathR.ModuleS&#45;&gt;PathI.ModuleJ</title>
<path fill="none" stroke="black" d="M103.32,-72.05C97.72,-63.89 90.88,-53.91 84.65,-44.82"/>
<polygon fill="black" stroke="black" points="87,-43.06 78.46,-36.79 81.23,-47.02 87,-43.06"/>
</g>
</g>
</svg>
