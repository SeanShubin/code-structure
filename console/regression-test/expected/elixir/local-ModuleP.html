<html>
  <head>
    <title>
      local-ModuleP
    </title>
    <link rel="stylesheet" href="reset.css">
    </link>
    <link rel="stylesheet" href="code-structure.css">
    </link>
  </head>
  <body>
    <h1>
      local-ModuleP
    </h1>
    <p>
      <a href="index.html">
        Table Of Contents
      </a>
    </p>
    <p>
      <a href="group.html">
        Group
      </a>
    </p>
    <p>
      <a href="code-units.html">
        Code Units
      </a>
    </p>
    <p>
      <a href="https://github.com/SeanShubin/code_structure_sample_elixir/blob/master/lib/module_p.ex">
        Source
      </a>
    </p>
    <div>
      <?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 12.1.2 (20240928.0832)
 -->
<!-- Title: detangled Pages: 1 -->
<svg width="197pt" height="260pt"
 viewBox="0.00 0.00 197.06 260.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 256)">
<title>detangled</title>
<polygon fill="lightgray" stroke="none" points="-4,4 -4,-256 193.06,-256 193.06,4 -4,4"/>
<!-- ModuleP -->
<g id="node1" class="node">
<title>ModuleP</title>
<g id="a_node1"><a xlink:href="local-ModuleP.html" xlink:title="ModuleP">
<ellipse fill="none" stroke="black" stroke-width="2" cx="69.78" cy="-234" rx="45.21" ry="18"/>
<text text-anchor="middle" x="69.78" y="-228.95" font-family="Times,serif" font-size="14.00" fill="blue">ModuleP</text>
</a>
</g>
</g>
<!-- PathP.ModuleQ -->
<g id="node3" class="node">
<title>PathP.ModuleQ</title>
<g id="a_node3"><a xlink:href="local-PathP-ModuleQ.html" xlink:title="PathP.ModuleQ">
<ellipse fill="none" stroke="black" cx="69.78" cy="-162" rx="69.78" ry="18"/>
<text text-anchor="middle" x="69.78" y="-156.95" font-family="Times,serif" font-size="14.00" fill="blue">PathP.ModuleQ</text>
</a>
</g>
</g>
<!-- ModuleP&#45;&gt;PathP.ModuleQ -->
<g id="edge1" class="edge">
<title>ModuleP&#45;&gt;PathP.ModuleQ</title>
<path fill="none" stroke="black" d="M69.78,-215.7C69.78,-208.41 69.78,-199.73 69.78,-191.54"/>
<polygon fill="black" stroke="black" points="73.28,-191.62 69.78,-181.62 66.28,-191.62 73.28,-191.62"/>
</g>
<!-- PathI.ModuleJ -->
<g id="node2" class="node">
<title>PathI.ModuleJ</title>
<g id="a_node2"><a xlink:href="local-PathI-ModuleJ.html" xlink:title="PathI.ModuleJ">
<ellipse fill="none" stroke="black" cx="69.78" cy="-18" rx="65.68" ry="18"/>
<text text-anchor="middle" x="69.78" y="-12.95" font-family="Times,serif" font-size="14.00" fill="blue">PathI.ModuleJ</text>
</a>
</g>
</g>
<!-- PathI.ModuleJ&#45;&gt;PathP.ModuleQ -->
<g id="edge2" class="edge">
<title>PathI.ModuleJ&#45;&gt;PathP.ModuleQ</title>
<path fill="none" stroke="black" d="M57.05,-36.15C50.49,-46.08 43.17,-59.12 39.78,-72 35.7,-87.47 35.7,-92.53 39.78,-108 42.16,-117.06 46.49,-126.19 51.13,-134.27"/>
<polygon fill="black" stroke="black" points="48.02,-135.89 56.25,-142.56 53.97,-132.21 48.02,-135.89"/>
</g>
<!-- PathR.ModuleS -->
<g id="node4" class="node">
<title>PathR.ModuleS</title>
<g id="a_node4"><a xlink:href="local-PathR-ModuleS.html" xlink:title="PathR.ModuleS">
<ellipse fill="none" stroke="black" cx="118.78" cy="-90" rx="70.29" ry="18"/>
<text text-anchor="middle" x="118.78" y="-84.95" font-family="Times,serif" font-size="14.00" fill="blue">PathR.ModuleS</text>
</a>
</g>
</g>
<!-- PathP.ModuleQ&#45;&gt;PathR.ModuleS -->
<g id="edge3" class="edge">
<title>PathP.ModuleQ&#45;&gt;PathR.ModuleS</title>
<path fill="none" stroke="black" d="M81.64,-144.05C87.3,-135.97 94.19,-126.12 100.5,-117.11"/>
<polygon fill="black" stroke="black" points="103.29,-119.23 106.16,-109.03 97.55,-115.21 103.29,-119.23"/>
</g>
<!-- PathR.ModuleS&#45;&gt;PathI.ModuleJ -->
<g id="edge4" class="edge">
<title>PathR.ModuleS&#45;&gt;PathI.ModuleJ</title>
<path fill="none" stroke="black" d="M106.91,-72.05C101.26,-63.97 94.36,-54.12 88.05,-45.11"/>
<polygon fill="black" stroke="black" points="91,-43.21 82.4,-37.03 85.26,-47.23 91,-43.21"/>
</g>
</g>
</svg>

    </div>
  </body>
</html>
